gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
I think that you don't need to specify java part, but still need to put src/main/java to scala source. 

But regarding the very first comment, I'm trying to use Scala classes from java source dir and vice versa and it seems to see each other